compiler/tools/suggest.nim

```diff
@@ -489,7 +602,7 @@ proc executeCmd*(cmd: IdeCmd, file, dirtyfile: AbsoluteFile, line, col: int;
 
   if dirtyfile.isEmpty: msgs.setDirtyFile(conf, dirtyIdx, AbsoluteFile"")
   else: msgs.setDirtyFile(conf, dirtyIdx, dirtyfile)
-
+  if cmd == ideOutline: return
```

Comment:
I think it'd be better if the logic above is moved into a separate procedure (e.g., `markDirtyFile`), with `executeCmd` only being called if `cmd` is not `ideOutline`.